requirements/edx/base.txt

```diff
@@ -771,7 +775,7 @@ ora2==4.5.1
     # via -r requirements/edx/base.in
 oscrypto==1.3.0
     # via snowflake-connector-python
-outcome-surveys==1.1.1
+outcome-surveys==2.0.0
```

Comment:
@muhammad-ammar I can see that you released this package. Is it safe to upgrade?

Reply:
@aht007 Yes.